modules/custom/az_search_api/config/install/search_api.index.az_search_api_index.yml

```diff
@@ -0,0 +1,307 @@
+    datasource_id: 'entity:node'
+    property_path: 'field_az_person_category_sec:entity:name'
+    type: string
+    dependencies:
+      config:
+        - field.storage.node.field_az_person_category_sec
+      module:
+        - taxonomy
+  field_az_short_title:
+    label: 'Short Title'
+    datasource_id: 'entity:node'
+    property_path: field_az_short_title
+    type: string
+    dependencies:
+      config:
+        - field.storage.node.field_az_short_title
+  field_az_subheading:
+    label: Subtitle
+    datasource_id: 'entity:node'
+    property_path: field_az_subheading
+    type: text
+    dependencies:
+      config:
+        - field.storage.node.field_az_subheading
+  field_az_summary:
+    label: Summary
+    datasource_id: 'entity:node'
+    property_path: field_az_summary
+    type: text
+    dependencies:
+      config:
+        - field.storage.node.field_az_summary
+  rendered_item:
+    label: 'Rendered HTML output'
+    property_path: rendered_item
+    type: text
+    configuration:
+      roles:
+        - anonymous
+      view_mode:
+        'entity:node':
+          ':default': search_index
+          az_event: ':default'
+          az_flexible_page: ':default'
+          az_news: ':default'
+          az_person: ':default'
+  title:
+    label: Title
+    datasource_id: 'entity:node'
+    property_path: title
+    type: string
+    dependencies:
+      module:
+        - node
+  type:
+    label: 'Content type'
+    datasource_id: 'entity:node'
+    property_path: type
+    type: string
+    dependencies:
+      module:
+        - node
+  url:
+    label: URI
+    property_path: search_api_url
+    type: string
+    configuration:
+      absolute: true
+datasource_settings:
+  'entity:node':
+    bundles:
+      default: false
+      selected:
+        - az_event
+        - az_flexible_page
+        - az_news
+        - az_person
+    languages:
+      default: true
+      selected: {  }
+processor_settings:
+  add_url: {  }
+  aggregated_field: {  }
+  auto_aggregated_fulltext_field: {  }
+  az_image_derivative:
+    weights:
+      preprocess_index: -20
+    all_fields: 0
+    fields:
+      field_az_media_thumbnail_image: field_az_media_thumbnail_image
+      field_az_awards: 0
+      field_az_body: 0
+      field_az_degrees: 0
+      field_az_event_category: 0
+      field_az_link: 0
+      field_az_location: 0
+      field_az_news_tags: 0
+      field_az_page_category: 0
+      field_az_person_category: 0
+      field_az_person_category_sec: 0
+      field_az_short_title: 0
+      field_az_subheading: 0
+      field_az_summary: 0
+      title: 0
+      type: 0
+      url: 0
+    image_style: az_enterprise_thumbnail_small
+  custom_value: {  }
+  entity_status: {  }
+  entity_type: {  }
+  language_with_fallback: {  }
+  rendered_item: {  }
+  solr_date_range:
+    weights:
+      preprocess_index: 0
+tracker_settings:
+  default:
+    indexing_order: fifo
+options:
+  cron_limit: 100
```

Comment:
I guess we need to decide if we want to preemptively set this value to something lower or just leave that as something to be fine-tuned on a site-by-site basis (depending on whether multiple search modules are installed that do indexing on cron runs).

Reply:
- [ ] Determine whether we wish to update the cron limits.